app/mailers/user_mailer.rb

```diff
@@ -7,6 +7,8 @@ def email_confirmation(user)
 
   def request_owner_access(user)
     @user = user
-    mail(to: 'wedding.salon777@gmail.com', subject: 'Запрос прав владельца')
+    mail(to: 'ENV["SENDMAIL_USERNAME"]', subject: 'New owner access request') do |format|
```

Comment:
SENDMAIL_USERNAME is not a correct name. This variable should store EMAIL and not USERNAME. It will be better to rename it to something like SYSTEM_EMAIL or something like this.